content/docs/command-reference/fetch.md

```diff
@@ -249,15 +248,14 @@ $ dvc status -c
     deleted:            model.pkl
     deleted:            data/features/test.pkl
     deleted:            data/features/train.pkl
-    deleted:            data/data.xml
```

Comment:
@jorgeorpinel When I replicated this example it didn't show `data/data.xml`.

Reply:
OK